validator/rpc/auth_token.go

```diff
@@ -136,52 +141,64 @@ func logValidatorWebAuth(validatorWebAddr, token string, tokenPath string) {
 			"the Prysm web interface",
 	)
 	log.Info(webAuthURL)
-	log.Infof("Validator CLient JWT for RPC and REST authentication set at:%s", tokenPath)
+	log.Infof("Validator CLient Auth Token for RPC and REST authentication set at:%s", tokenPath)
```

Comment:
I'm not confident in these logs, it provides a good user experience to be easily clickable but not sure how safe it is to put it here?

Reply:
`CLient` ==> `Client`